tests/test.sh

```diff
@@ -364,17 +369,29 @@ run_ice()
         fi
     fi
 
-    remove_debug_info="s/DW_AT_\(GNU_dwo_\(id\|name\)\|comp_dir\|producer\|linkage_name\|name\).*//g"
+    local remove_offset_number='s/<[A-Fa-f0-9]*>/<>/g'
+    local remove_debug_info="s/\(Length\|DW_AT_\(GNU_dwo_\(id\|name\)\|comp_dir\|producer\|linkage_name\|name\)\).*/\1/g"
```

Comment:
Added "Length" as an additional field to get removed. Also updated this so that only the value is deleted and the name is left alone. e.g. before it was making the change "<offset>   DW_AT_comp_dir : /some/path  --> <offset>" now it is "<offset>   DW_AT_comp_dir : /some/path  --> <offset> DW_AT_comp_dir"